client/lib/feed-stream-store/paged-stream.js

```diff
@@ -65,8 +64,6 @@ export default class PagedStream {
 			return;
 		}
 
-		Dispatcher.waitFor( [ FeedPostStore.dispatchToken ] );
```

Comment:
flux has a notion of store dependencies. so you can wait until one store has handled an action before allowing other stores to process the same action. 

we used to wait on the feed post store to process an action because we would then use the feed post store while processing our actions. this would prevent the feed-stream-store from asking for a post before it was present in the feed post store.

since we no longer really depend on the feed post store for the date / ordering property, we no longer have to wait for it.